docs/src/utils/getComponentPathname.ts

```diff
@@ -3,9 +3,9 @@ import * as _ from 'lodash'
 /**
  * Returns a pathname for a given component.
  *
- * @param {object} info - A Component's *.info.json
- * @returns {string}
+ * @param info - A Component's *.info.json
```

Comment:
Out of curiosity, is there any reason some of the JSDoc types were removed and others updated?

Reply:
All of them were removed I thought. And if they weren't already in the actual TS signatures, I added them there.